ale/drivers/__init__.py

```diff
@@ -51,19 +51,18 @@ def load(label):
                String path to the given label file
     """
     for name, driver in drivers.items():
-            print("Trying:", name)
-            try:
-                res = driver(label)
-                if res.is_valid():
-                    with res as r:
-                            return res.to_dict()
-            except Exception as e:
-                import traceback
-                print("Driver Failed:", e)
-                traceback.print_exc()
+        print(f'Trying {name}')
+        try:
+            res = driver(label)
+            with res as driver:
+                return formater(driver)
+        except Exception as e:
+            import traceback
```

Comment:
Why not just import above?

Reply:
This was in the function originally, I can move it with the others though